HPXMLtoOpenStudio/resources/meta_measure.rb

```diff
@@ -332,6 +367,11 @@ def run_measure(model, measure, argument_map, runner)
       runner.registerFinalCondition(result_child.finalCondition.get.logMessage)
     end
 
+    # re-register runner child registered values on the parent runner
+    result_child.stepValues.each do |step_value|
+      runner.registerValue(step_value.name, get_value_from_workflow_step_value(step_value))
+    end
+
```

Comment:
@shorowit This is needed for ResStockArguments to work. It doesn't break anything in the OS-HPXML, but should we make it optional anyway?

Reply:
No, I don't think it needs to be optional.